src/pages/LoginForm.js

```diff
@@ -0,0 +1,170 @@
+import React, { useState } from "react";
+import {
+  Flex,
+  Box,
+  Heading,
+  FormControl,
+  FormLabel,
+  Input,
+  Button,
+  CircularProgress,
+  Text,
+  Link,
+} from "@chakra-ui/core";
+
+import ErrorMessage from "../components/ErrorMessage";
+
+const LoginForm = () => {
+  const [state, setState] = useState({
+    email: "",
+    password: "",
+    isLoading: false,
+    error: "",
+    signUp: true,
+  });
+  const { email, confirmPassword, password, isLoading, error, signUp } = state;
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    alert(`Email: ${email} & Password: ${password}`);
+  };
+
+  const handleChange = (event) => {
+    const { value, name } = event.target;
+
+    setState({ ...state, [name]: value });
+  };
+
+  const switchMode = () => {
```

Comment:
Can we change the name to something else, `switchMode` sounds like a toggle for a dark and light mode. Maybe `toggleSignupLogin`?